animation_nodes/algorithms/mesh_generation/line.pyx

```diff
@@ -33,10 +41,15 @@ def vertices(start, end, long steps):
 # Edges
 ############################################
 
-def edges(long steps):
-    edges = EdgeIndicesList(length = steps - 1)
+def edges(long steps, bint cyclic):
+    cdef long edgeAmount
+    edgeAmount = steps if cyclic else steps - 1
```

Comment:
You can put the initialization in the declaration directly.

Reply:
Modified.